frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -810,6 +810,50 @@ describe("scenarios > visualizations > pivot tables", () => {
         .click();
     }
   });
+
+  it("should not show subtotals for flat tables", () => {
```

Comment:
This test ensures subtotals are not displayed if the top-level rows are flat (containing only one child). That's not a new behavior, it exists on `master` already.

Correct             |  Broken
:-------------------------:|:-------------------------:
<img width="823" alt="before-fix" src="https://user-images.githubusercontent.com/17258145/121926365-91a75700-cd46-11eb-972c-66b41c2fe6c5.png">  |  <img width="823" alt="after-fix" src="https://user-images.githubusercontent.com/17258145/121926385-953ade00-cd46-11eb-8c6e-21ecfe9b49e3.png">